STUDENT_COURSE _MANAGEMENT/src/main/resources/templates/access-denied.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Access Denied</title>
+</head>
+
+<body>
+
+<h2>Access Denied - You are not authorized to access this resource</h2>
+
+<hr>
+
+<a th:href="@{/students/list}" class="btn btn-primary btn-sm mb-3" >
```

Comment:
from where this class is being imported?

Reply:
I forgot to add link to bootstrap . I added it.